example/.gitignore

```diff
@@ -9,3 +9,4 @@ ios/.symlinks/
 packages
 pubspec.lock
 .flutter-plugins
+*.iml
```

Comment:
use a [global .gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for your flavor of IDE instead

Reply:
Ha, I learned something new and very cool. Thanks. Will do.